bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -26,6 +27,8 @@ parser.add_argument('--signal-file', help='File with parameters of GW signals '
                     'for KDE calculation')
 parser.add_argument('--template-file', required=True, help='Hdf5 file with '
                     'template masses and spins')
+parser.add_argument('--attr-filename', required=True, help='Saved as an attr '
```

Comment:
I am not seeing what this should be used for. Can you give an example of what you might give for this option?  Why is it called 'filename' ?

Reply:
This attr stores the file name used in the signal-kde or template-kde output file. 
for example, in the latest template-kdevals.hdf, this attr gives "sbbh2_Op985.hdf"
& in case of signal-kdevals.hdf, this attr gives "4OGC"

Reply:
ok, but I am not seeing why this needs to be a command line option at all if the signal file or template file names are already options. Can you just set the attr to the actual name of the input file?